File: GitHub/Math/ProjectEuler/Python/Euler2.py
limit = int(input())
old_fib = 0
new_fib = 1
fib_sum = 0
while new_fib < limit:
	new_fib = new_fib + old_fib
	old_fib = new_fib - old_fib
	if new_fib % 2 == 0:
		fib_sum += new_fib
print(fib_sum)

File: GitHub/Math/ProjectEuler/Java/Euler2.java
class Euler2{
	public static void main(String[] args){
		int oldfib = 1;
		int newfib = 1;
		int sum = 0;
		while(newfib < 4000000){
			newfib = oldfib + newfib;
			oldfib = newfib - oldfib;
			if(newfib % 2 == 0){
				sum += newfib;
			}
		}
		System.out.println(sum);
	}
}
